1999-12-13  Jakub Jelinek  <jakub@redhat.com>

	* streambuf.h (streampos): Typedef to _IO_off_t resp. _IO_off64_t.
	* parsestream.h (class parsebuf::pos_at_line_start): Change type
	to _IO_off_t.

*** /usr/include/g++/streambuf.h.jj	Sun Aug  8 20:07:01 1999
--- /usr/include/g++/streambuf.h	Mon May 15 16:04:27 2000
***************
*** 72,81 ****
  
  #if defined(_G_IO_IO_FILE_VERSION) && _G_IO_IO_FILE_VERSION == 0x20001
  typedef _IO_off64_t streamoff;
! typedef _IO_fpos64_t streampos;
  #else
  typedef _IO_off_t streamoff;
! typedef _IO_fpos_t streampos;
  #endif
  typedef _IO_ssize_t streamsize;
  
--- 72,81 ----
  
  #if defined(_G_IO_IO_FILE_VERSION) && _G_IO_IO_FILE_VERSION == 0x20001
  typedef _IO_off64_t streamoff;
! typedef _IO_off64_t streampos;
  #else
  typedef _IO_off_t streamoff;
! typedef _IO_off_t streampos;
  #endif
  typedef _IO_ssize_t streamsize;
  
*** /usr/include/g++/parsestream.h.jj	Thu Aug 21 18:58:20 1997
--- /usr/include/g++/parsestream.h	Mon May 15 16:04:27 2000
***************
*** 42,48 ****
  
  class parsebuf : public streambuf {
    protected:
!     _IO_fpos_t pos_at_line_start;
      long _line_length;
      unsigned long __line_number;
      char *buf_start;
--- 42,48 ----
  
  class parsebuf : public streambuf {
    protected:
!     _IO_off_t pos_at_line_start;
      long _line_length;
      unsigned long __line_number;
      char *buf_start;
